profiles/sssd/system-auth

```diff
@@ -19,6 +19,7 @@ auth        sufficient                                   pam_sss.so forward_pass
 auth        required                                     pam_faillock.so authfail                               {include if "with-faillock"}
 auth        optional                                     pam_gnome_keyring.so only_if=login auto_start          {include if "with-pam-gnome-keyring"}
 auth        required                                     pam_deny.so
+-auth       sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
For some reason this has to be above the `pam_deny.so` block in the `auth` section (also in the other file).